src/main/scala/dpla/ingestion3/JsonlEntry.scala

```diff
@@ -0,0 +1,59 @@
+package dpla.ingestion3
+
+import dpla.ingestion3.model.{DplaMapData, ModelConverter, jsonlRecord}
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.{Dataset, SparkSession}
+
+/**
+  * Driver for reading enriched data and generating JSONL text, which can be
```

Comment:
Correct, it would technically work with any Avro having DplaMap records. Would we want to load unenriched data into the search index? I guess for now we're doing when we use the legacy QA system for Ingestion3 QA. I'll change this to mention DplaMap in general.